qualification/tied_array_channelised_voltage/test_linearity.py

```diff
@@ -70,12 +70,12 @@ async def _test_linearity(
     assert scales[middle] == pytest.approx(1.0)
     powers = np.zeros_like(scales)
     for batch in batched(enumerate(scales), len(receiver.stream_names)):
-        for (_, scale), beam_name in zip(batch, receiver.stream_names):
+        for (_, scale), beam_name in zip(batch, receiver.stream_names, strict=False):
             await set_variable(client, beam_name, scale)
             pdf_report.detail(f"Set {variable} to {scale} on {beam_name}.")
         _, data = await receiver.next_complete_chunk()
         pdf_report.detail("Received chunk.")
-        for (i, _), d, beam_name in zip(batch, data, receiver.stream_names):
+        for (i, _), d, beam_name in zip(batch, data, receiver.stream_names, strict=False):
```

Comment:
Again, comment please.

Reply:
Haven't added a separate one here - the one I've just added at the top of the for loop should cover both.